study-builder/studies/brugada/snippets/picklist6.conf

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
can we have some meaningful file name that reflects what these options are rather than picklist6.conf

Reply:
renamed